chess engine/scores.csv

```diff
@@ -0,0 +1,376 @@
+Id, Pgn Number, Event, Move Number, Move, Score, Comment
```

Comment:
I'm not sure if we should check this file into the repository. We could upload it to the drive and keep in here only the source code

Reply:
file deleted accordingly